### ISSUE TITLE
Error in pushing committed files to SCM

### ISSUE THREAD

**Issue by yousafsaqib** [scm-manager/scm-manager#1252]
I am facing issue while pushing my committed files on main branch. Pull is working fine but pushing files 

SCM-Manager is open source. If you have any knowledge of programming, maybe you can help. We would love to support you.

M using Windows 10 and using git 1.9.5

below is the server log

git -c diff.mnemonicprefix=false -c core.quotepath=false --no-optional-locks push -v --tags origin master:master
POST git-receive-pack (527 bytes)
remote: error: [SCM] value is empty
Pushing to http://pdc:8080/scm/repo/Marketing/myproject.com
To http://pdc:8080/scm/repo/Marketing/myproject.com
 = [up to date]          myproject_Website_Release -> myproject_Website_Release
 = [up to date]          Before_Freeze -> Before_Freeze
 = [up to date]          Only_this_file_made_live_as_being_urgent -> Only_this_file_made_live_as_being_urgent
 = [up to date]          Pre-Mobile-Friendly -> Pre-Mobile-Friendly
 = [up to date]          Release_Dated_07-09-2015 -> Release_Dated_07-09-2015
 ! [remote rejected]     master -> master (unspecified reason)
error: failed to push some refs to 'http://pdc:8080/scm/repo/Marketing/myproject.com'

below is the error that i get while pushing files from terminal

ounting objects: 24, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (5/5), done.
Writing objects: 100% (5/5), 509 bytes | 0 bytes/s, done.
Total 5 (delta 4), reused 0 (delta 0)
remote: Resolving deltas: 100% (4/4)
remote: error: [SCM] value is empty
To http://yousaf_saqib@pdc:8080/scm/repo/myproject/myproject.com
 ! [remote rejected] master -> master (unspecified reason)
error: failed to push some refs to 'http://yousaf_saqib@pdc:8080/scm/repo/myproject/myproject.com'

**Comment by eheimbuch** [comment]
Hey yousafsaqib,

it seems like a commit of yours have a empty message. You could try to modify this commit and push again.

**Comment by yousafsaqib** [comment]
@eheimbuch ,
that's not the issue, i have written a detailed description about commit then committed the files. My teammate facing the same issue [not able to push].

**Comment by eheimbuch** [comment]
@yousafsaqib That's quite strange. I would guess that your repository config somehow got messed up. Could you try to create a new repository in SCM-Manager and push your changes there? Does this error occur on a single repository or even a specific branch only? Also please share your SCM-Manager version with us.

**Comment by yousafsaqib** [comment]
@eheimbuch Yes, i am facing this problems for multiple repos. And i have created a new repo, with only one branch and got same problem. I am using the latest version of scm-manager (2.2.0), with Java 11.0.7 and git version is 2.27.0.windows.1 on my git server.

**Comment by eheimbuch** [comment]
@yousafsaqib Thanks. Could you please check if your repository config contains an `[scmm]` Entry?

You should find it here (use your repository id):
`cat ~/scm-home/repositories/BpS4tMWjR2/data/config` 

It should like somewhat like this:
```
[core]
        repositoryformatversion = 0
        filemode = true
        bare = true
        logallrefupdates = false
[scmm]
        repositoryid = BpS4tMWjR2```
